www/src/theme/SearchBar/index.js

```diff
@@ -12,6 +12,12 @@ import { useHistory } from "@docusaurus/router"
 import useDocusaurusContext from "@docusaurus/useDocusaurusContext"
 import "./styles.css"
 
+const kFormatter = (num) => {
+  return Math.abs(num) > 999
+    ? Math.sign(num) * (Math.abs(num) / 1000).toFixed(1) + "k"
+    : Math.sign(num) * Math.abs(num)
```

Comment:
hopefully we will never get below 1000, so I'm theory this logic could be simplified. 😁

Reply:
Hahah good call. Cleaned it up :+1: